docs/user-guide/cli-using-using-daemon-mode.md

```diff
@@ -50,6 +50,27 @@ The following example illustrates running Zowe CLI commands with the daemon mode
    ```
 **Note:** When you are running Zowe CLI in daemon using a Git Bash terminal on a Windows operating system, special characters might display using the wrong code page. For example, the default code page 437 renders a form-feed character (\f) as an emoji (♀️). To correct the problem, issue the command `chcp.com 65001` to change the code page to UTF-8. If you want the change to be persistent, add the command to your .bashrc file.
 
+## Restart daemon mode
+
+Daemon mode is a long-running background process (waits for work) that significantly improves Zowe CLI performance. When you make changes to your work environment, daemon mode does not capture the changes. Restarting daemon mode lets the daemon capture the changes. Issue the following command to stop the currently running daemon and start a new daemon:
+
+```
+Zowe daemon restart
+```
+
+Restart the daemon under the following scenarios:
+
+- When you change the value of any of the following [Zowe CLI environment variables](../user-guide/cli-configuringcli.md):
+  - `ZOWE_CLI_HOME`
+  - `ZOWE_APP_LOG_LEVEL`
+  - `ZOWE_IMPERATIVE_LOG_LEVEL`
+- After you install, update, or uninstall a plug-in.
+- After you install a newer version of Zowe CLI and issue the `zowe daemon enable` command. (You can install a newer version of CLI and issue the command while daemon mode is running.)
```

Comment:
In this bullet, I think that we want to convey that if you install a new version of CLI, you ***HAVE TO*** enable the daemon again, and if a daemon was already running, you ***HAVE TO*** restart the daemon.